roles/wisdom/wisdom_llm_load_test_multiplexed/templates/llm-load-test-config.json.j2

```diff
@@ -0,0 +1,52 @@
+{
+  "general": {
+    "schema_version": 1.0,
+    "debug": true
+  },
+  "storage": {
+    "type": "s3",
+    "s3_params": {
+      "s3_host": "",
+      "s3_use_https": true,
+      "s3_access_key": "",
+      "s3_secret_key": "",
+      "s3_bucket": "wisdom-perf-data-test",
+      "s3_region": "us-east-1",
+      "s3_separator": "/",
+      "s3_base_path": "InferenceResults",
+      "s3_component": "ModelMesh",
+      "s3_sub_component": "WatsonRuntime_v0.27.0",
+      "s3_sub_comp_version": "Wisdom_v11"
+    }
+  },
+  "launcher": {
+    "type": "ghz",
+    "ghz_params": {
+      "host": "modelmesh-serving.wisdom.svc:8033",
+      "insecure": true,
+      "proto_path": "/etc/load_test_protos/wisdomextservice.proto",
+      "call": "caikit.runtime.WisdomExt.WisdomExtService/CodeGenerationTaskPredict",
+      "vmodel_id": "ansible-wisdom",
+      "query": "",
+      "context": "",
+      "concurrency": {{ wisdom_llm_load_test_multiplexed_concurrency }},
+      "requests": {{ wisdom_llm_load_test_multiplexed_requests }},
+      "max_duration": {{ wisdom_llm_load_test_multiplexed_max_duration }},
+    },
+    "input_dataset": {
+      "filename": "/etc/load_test_dataset/dataset.json",
+      "max_size": 3
+    }
+  },
+  "warmup": false,
+  "test_conditions": {
+    "platform_type": "ocp",
+    "modelmesh_pods_per_node": 1,
+    "max_batch_size": 16,
+    "nodes": {{ wisdom_llm_load_test_multiplexed_replicas }},
+    "instance_type": "g5.2xlarge",
+    "instance_gpu_count": 1,
```

Comment:
I would move those to `config.yaml`,
maybe in `test.py`, extract a subset of `config.yaml`, store it in `$ARTIFACT_DIR`, and pass the path to this role?
(to avoid passing too many parameters via the CLI)